package.json

```diff
@@ -26,7 +26,8 @@
         "eslint-plugin-no-only-tests": "2.3.1",
         "eslint-plugin-prettier": "3.1.2",
         "eslint-plugin-react": "7.16.0",
-        "eslint-plugin-react-hooks": "2.2.0"
+        "eslint-plugin-react-hooks": "2.2.0",
+        "prettier": "2.0.5"
```

Comment:
seems like this would be a major release though, as it means the consumers have to remove their prettier and also update accordingly

Reply:
Yes - this way we can standardize the formatting configuration further.
Prettier can be removed as a dependency from all consumers of this repository.